app/src/main/java/com/hadiyarajesh/notex/ui/settings/SettingsScreen.kt

```diff
@@ -0,0 +1,120 @@
+package com.hadiyarajesh.notex.ui.settings
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import com.hadiyarajesh.notex.R
+
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SettingsScreen(
+    navController: NavController
+) {
+    val settingsItems = listOf(
```

Comment:
This list will be created every time re-composition happens. Avoid that. we don't need this if we use `sealed class Screens`

Reply:
updated in the next PR maybe tomorrow.